app/platform/config.go

```diff
@@ -30,7 +35,145 @@ func (c *ServiceConfig) validate() error {
 	}
 
 	if c.Logger == nil {
-		return errors.New("Logger is required")
+		var err error
+		c.Logger, err = mlog.NewLogger()
+		if err != nil {
+			return err
+		}
```

Comment:
Hmm.. is this right? Why can't we pass the logger from above?

Reply:
That was just a temp step, I should've add a comment there. This will be fixed after https://mattermost.atlassian.net/browse/MM-45841

Reply:
Yep, let's add a comment to clarify.